### PR TITLE
Donate Feature & Code cleanup

### PR DESCRIPTION
**Pull request by pramathramesh**
Created a component folder called Donate, housing donate components rendered at the homepage and navbar pop up.

### CODE REVIEW

File: src/Layout/Pages/Home/LandingSlogan/LandingSlogan.jsx [hunk 1]
```diff
@@ -2,31 +2,32 @@ import SloganImg from '../../../../assets/Homepage/LandingImg.png';
...
                     </div>
                 </div>
             </div>
             <div className="landing-intro-right-container">
                 <img src={SloganImg} className="right-img" alt="right-img" />
-                <div className="slogan-desc">
+                {/* <div className="slogan-desc">
                     <div className="action-items">
-                        <FilledButton>Donate Now</FilledButton>
+                        <FilledButton label="hhhhhhhhhhhhhhhhhhhhh" />
```
Comment (roshan1999) [inline src/Layout/Pages/Home/LandingSlogan/LandingSlogan.jsx:24]: change this

**Comment by roshan1999** [comment]
Issues Identified: 
[ ] Overlap of donations section in the home page with team parent
[ ] Clicking of donate now button in section causes the button to still open razorpay window even if errors in the form. 
[ ] Sizing of the donate now button

File: src/Components/Donate/DonateDescriptoin.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
```
Comment (roshan1999) [inline src/Components/Donate/DonateDescriptoin.jsx:1]: Update the file name 

- [ ] DonateDescriptoin  to DonateDescription

File: src/Components/Donate/Donate.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import './Donate.css'
+import DonateBox from "./DonateBox";
+import DonateDescription from "./DonateDescriptoin";
+
+
+const whyDonate = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.";
```
Comment (roshan1999) [inline src/Components/Donate/Donate.jsx:7]: - [ ] Update content

File: src/Layout/Pages/Home/LandingSlogan/LandingSlogan.jsx (hunk 1 above)
Comment (pramathramesh) [reply src/Layout/Pages/Home/LandingSlogan/LandingSlogan.jsx:24]: This section is not being used anywhere on the website. I'll be removing this commented section.

File: src/Components/Donate/Donate.jsx (hunk 3 above)
Comment (pramathramesh) [reply src/Components/Donate/Donate.jsx:7]: I have requested the content. Will update.

**Pull request merged**